src/maven-repair/pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>org.apache.maven.surefire</groupId>
             <artifactId>surefire-report-parser</artifactId>
-            <version>2.20</version>
```

Comment:
We can move the changes in .ci/ci-maven-repair.sh and src/maven-repair/pom.xml in a separate baby PR to be merged first, to slim this down.

Reply:
This breaks maven-repair tho, because the current parsing of the report in NPEFixMojo is not compatible with 2.21.

I had to upgrade it because repairnator-pipeline uses 2.21.

Reply:
OK, we leave it there then